libs/providers/flagd/README.md

```diff
@@ -38,6 +38,7 @@ Options can be defined in the constructor or as environment variables. Construct
 | selector                               | FLAGD_SOURCE_SELECTOR          | string  | -                                                              |                  |
 | cache                                  | FLAGD_CACHE                    | string  | lru                                                            | lru, disabled    |
 | maxCacheSize                           | FLAGD_MAX_CACHE_SIZE           | int     | 1000                                                           |                  |
+| defaultAuthority                       | FLAGD_DEFAULT_AUTHORITY        | string  | -                                                              | rpc, in-process  |
```

Comment:
Reminder to myself to include this configuration option in the flagd provider spec.

https://flagd.dev/reference/specifications/providers/#configuration

Reply:
I'm already making some flagd doc changes, I can do it.